### PR TITLE
Added build script, build info and dockerfile for freemarker

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: a/apache-freemarker/Dockerfiles/ubi_8.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8:8.5
+
+ARG PACKAGE_VERSION=v2.3.32
+
+RUN yum install -y sudo \
```
Comment (kandarpamalipeddi) [inline a/apache-freemarker/Dockerfiles/ubi_8.5/Dockerfile:5]: Do we need `sudo` ?
As we are not setting the user using the `user` command.

Also, Can we club all the`RUN` commands to single one?
Also, after extracting tars remove them.
That saves some amount of Docker image size.

File: a/apache-freemarker/apache-freemarker_v2.3.31_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+ant download-ivy
+
+if ! ant; then 
+	echo "Build fails"
+	exit 1
+fi
+
+
+echo "1i boot.classpath.j2se1.8=/usr/lib/jvm/jre/lib/rt.jar" > build.properties
```
Comment (kandarpamalipeddi) [inline a/apache-freemarker/apache-freemarker_v2.3.31_ubi_8.5.sh:52]: Just to confirm, we are creating `build.properties` file.
Am I correct?

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: a/apache-freemarker/apache-freemarker_v2.3.31_ubi_8.5.sh (hunk 2 above)
Comment (muskaan2199) [reply a/apache-freemarker/apache-freemarker_v2.3.31_ubi_8.5.sh:52]: @kandarpamalipeddi yes

File: a/apache-freemarker/Dockerfiles/ubi_8.5/Dockerfile (hunk 1 above)
Comment (muskaan2199) [reply a/apache-freemarker/Dockerfiles/ubi_8.5/Dockerfile]: @kandarpamalipeddi made the changes

**Comment by muskaan2199** [comment]
http://129.40.81.8:8080/job/MN_Loading_json_build/203/

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**